funi.js

```diff
@@ -19,18 +19,17 @@ const m3u8 = require('m3u8-parsed');
 const streamdl = require('hls-download');
 
 // extra
-const modulesFolder = __dirname + '/modules';
```

Comment:
I would do the same thing here as you did in line 29 (old) or 28 (new)

Reply:
Yes, I should have. Part of the code is taken from crunchyroll downloader, I just didn't notice.